app/javascript/views/Login/Login.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import Item from "@mui/material/Grid";
+import LoginIcon from "@mui/icons-material/Login";
+import Stack from "@mui/material/Stack";
+import TextField from "@mui/material/TextField";
+
+import "./Login.scss";
+const Login = () => {
+  const [name, setName] = React.useState();
+  const [password, setPassword] = React.useState();
+  
+  const [nameError, setNameError] = React.useState();
+  const [passwordError, setPasswordError] = React.useState();
+  
+  const [isSubmitting, setIsSubmitting] = React.useState(false);
+
+  const handleLoginSubmit = () => {
+    setIsSubmitting(true);
+    clearPreviousErrors();
+
+    if (!validateForm()) {
+      setIsSubmitting(false);
+      return;
+    }
+
+    // TODO: Submit POST request to /api/v1/users/login
+    setTimeout(() => {
+      setIsSubmitting(false);
+    }, 3000);
+  };
+
+  const validateForm = () => {
+    if (!name) setNameError("Name is required");
+    if (!password) setPasswordError("Password is required");
+    return name && password ? true : false;
+  };
```

Comment:
Idea: explore some front-end validation libraries to make validating forms faster and require less manual work?